cmd/zoekt-sourcegraph-indexserver/merge.go

```diff
@@ -12,13 +12,21 @@ import (
 	"time"
 
 	"github.com/google/zoekt"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
 	"gopkg.in/natefinch/lumberjack.v2"
 )
 
 var reCompound = regexp.MustCompile(`compound-.*\.zoekt`)
+var shardMergingRunning = promauto.NewGauge(prometheus.GaugeOpts{
```

Comment:
How about we also capture how long merging takes, together with if it errored or not?